### DISCUSSION TITLE
about QR Code 64BT

### DISCUSSION THREAD

**Discussion by jh-martin-1** [gopro/labs#1029] [Q&A]
不知道什么原因，我之前的github账号被暂停了。
我之前的问题是：当我设置了64BT=1000或64BT=2000，black10都不能连续录像直到卡满（总会剩余一些卡容量），当我改成更大的分段章节（!M64BT=0)，它可以正常连续录像直到卡满。
请问，是否能够让labs更好的支持比如64BT=1000也可以正常连续录制直到卡满。谢谢！

**Comment by dnewman-gpsw** [comment]
Unfortunately not, this feature was designed to increase chapter size, for which it works well.  Decreasing the chapters also working well. Likely there is a test for card size remaining that is confused by the small chapter size, but most of the card is used. As this is Labs, and not production firmware, it is about experimentation and extending camera abilities as a prototype for potential future camera enhancements, not a necessarily a polished feature.

**Comment by jh-martin-1** [comment]
目前来说，Labs的这些功能基本上可以满足black10作为行车记录仪的使用，就是略微有些不便，比如太大的单个文件导出会比较麻烦和费时，又由于录制时USB电源不能同时给电池充电，这样需要人工关注电池的余量。期待能够解决64BT=1000（或者一个小的数字）的设置下能够连续录制直到卡满，同时最好能够提供一个选项，以便让用户选择卡满后是否自动删除最早的文件。还有一个需求是插入USB电源后，在录制时可以同时给电池充电。对于有些用户，行车记录仪功能是一个重要的需求之一。对于Labs的这些建议或者需求，在哪里提出呢？以便能够得到实现！谢谢您的回复！
